NuRadioReco/detector/RNO_G/db_mongo_read.py

```diff
@@ -0,0 +1,1048 @@
+import six
+import os
+import urllib.parse
+import datetime
+import numpy as np
+from functools import wraps
+import copy
+import re
+
+from pymongo import MongoClient
+import NuRadioReco.utilities.metaclasses
+import astropy.time
+
+import logging
+logging.basicConfig()
+logger = logging.getLogger("NuRadioReco.MongoDBRead")
+logger.setLevel(logging.INFO)
+
+
+def _convert_astro_time_to_datetime(time_astro):
+    if not isinstance(time_astro, astropy.time.Time):
+        raise ValueError("This function expects only arguments from type `astropy.time.Time`")
+
+    return time_astro.to_datetime()
```

Comment:
Seeing as the isinstance check is implemented before calling this - why have this function instead of just calling time_astro.to_datetime()? But okay, this is nitpicking.

Reply:
fixed